helpers/dynamic-fixtures-helper.js

```diff
@@ -0,0 +1,182 @@
+        this.http = http;
+    }
+
+    haveCustomersWithQuotes(customerCount, quoteCount = 1, itemCount = 10, defaultItemPrice = 1000) {
+        const defaultParams = {
+            headers: {
+                'Content-Type': 'application/vnd.api+json',
+            },
+        };
+
+        const dynamicFixturesResponse = this.http.sendPostRequest(
+            this.http.url`${this.backendApiUrl}/dynamic-fixtures`,
+            JSON.stringify(this._getCustomersWithQuotesAttributes(customerCount, quoteCount, itemCount, defaultItemPrice)),
+            defaultParams,
+            false
+        );
+
+        const dynamicFixturesResponseJson = JSON.parse(dynamicFixturesResponse.body);
+        const customerData = dynamicFixturesResponseJson.data.filter(item => /^customer\d+$/.test(item.attributes.key));
+
+        return customerData.map(customer => {
+            const associatedCustomerQuotes = dynamicFixturesResponseJson.data
+                .filter(item => item.attributes.key.startsWith(`${customer.attributes.key}Quote`))
+                .map(quote => quote.attributes.data.uuid);
+
+            return {
+                customerEmail: customer.attributes.data.email,
+                quoteIds: associatedCustomerQuotes
+            };
+        });
+    }
+
+    haveConsoleCommands(commands) {
+        const params = {
+            headers: {
+                'Content-Type': 'application/vnd.api+json',
+            },
+            timeout: 600000,
+        };
+
+        this.http.sendPostRequest(
+            this.http.url`${this.backendApiUrl}/dynamic-fixtures`,
+            JSON.stringify(this._getConsoleCommandsAttributes(commands)),
+            params,
+            false
+        );
+    }
+
+    _getConsoleCommandsAttributes(commands) {
+        const operations = commands.map((command) => {
+            return {
+                type: 'cli-command',
+                name: command,
+            };
+        });
+
+        return {
+            data: {
+                type: 'dynamic-fixtures',
+                attributes: {
+                    operations: operations,
+                },
+            },
+        };
+    }
+
+    _getCustomersWithQuotesAttributes(customerCount = 1, quoteCount = 1, itemCount = 10, defaultItemPrice = 100) {
+        const baseOperations = [
+            {
+                type: 'transfer',
+                name: 'LocaleTransfer',
+                key: 'locale',
+                arguments: { id_locale: 66, locale_name: 'en_US' }
+            },
+            {
+                type: 'transfer',
+                name: 'ProductImageTransfer',
+                key: 'productImage',
+                arguments: {
+                    externalUrlSmall: 'https://images.icecat.biz/img/gallery_mediums/30691822_1486.jpg',
+                    externalUrlLarge: 'https://images.icecat.biz/img/gallery/30691822_1486.jpg'
+                }
+            }
+        ];
+
+        // Generate products dynamically
```

Comment:
Minor: generate products, customers, etc would be better to move to a separate methods just for readability

Reply:
It should be reworked in next phase. For now I would keep it as it is.